### PR TITLE
make install-models: delegate to ocrd resmgr

### PR DESCRIPTION
**Pull request by kba**
Once OCR-D/core#559 is merged.

### CODE REVIEW

**Review by bertsky — CHANGES_REQUESTED** [review]
I recommend a more radical change: Instead of unconditionally delegating `install-models` to `install-models-ocropus`, `install-models-calamari` etc, we should make these subtargets depend on whether the respective module is in the currently active `OCRD_MODULES`, and encapsulate each such rule near the other rules for the modules. 

Thus, instead of …
```make
ifneq ($(findstring ocrd_cis, $(OCRD_MODULES)),)
OCRD_EXECUTABLES += $(OCRD_CIS)
...
```

…we would have…
```make
ifneq ($(findstring ocrd_cis, $(OCRD_MODULES)),)
install-models: install-models-cis
install-models-cis: $(BIN)/ocrd
	. $(ACTIVATE_VENV) && ocrd resmgr download ocrd-cis-ocropy-recognize '*'
OCRD_EXECUTABLES += $(OCRD_CIS)
...
```

Also, we still need to adapt `Dockerfile`:
```Dockerfile
ENV XDG_DATA_HOME /usr/local/share
VOLUME $XDG_DATA_HOME/ocrd-resources
```

(perhaps also `HOME` itself)

**Comment by kba** [comment]
Having thought about it some more, I agree that `install-models-*` should depend on `OCRD_MODULES` to ensure that only the models are installed for those processors the user requested (by having it in `OCRD_MODULES`)

**Comment by kba** [comment]
> we should make these subtargets depend on whether the respective module is in the currently active OCRD_MODULES

fb4ee26

> Also, we still need to adapt Dockerfile:

2fdce0a

File: Makefile [hunk 1]
```diff
@@ -384,6 +394,10 @@ $(OCRD_PAGETOPDF): ocrd_pagetopdf $(BIN)/ocrd
 endif
 
 ifneq ($(findstring ocrd_calamari, $(OCRD_MODULES)),)
+install-models: install-models-calamari
+.PHONY: install-models-calamari
+install-models-calamari: $(BIN)/ocrd
+	. $(ACTIVATE_VENV) && ocrd resmgr download ocrd-calamari-binarize '*'
```
Comment (bertsky) [inline Makefile:400]: Shouldn't that be…
```suggestion
	. $(ACTIVATE_VENV) && ocrd resmgr download ocrd-calamari-recognize '*'
```
…actually?

**Review by bertsky — APPROVED** [review]
Splendid – thanks!

Perhaps we could offer more of these `install-models-*` rules in order to allow a quick installation without studying all processors and available models? (Currently, I would still have to `ocrd resmgr download PROCESSOR '*'` selectively for each processor I might want to use.)

**Comment by kba** [comment]
> Perhaps we could offer more of these `install-models-*`

384ee6d I added targets for all projects that we have in core's `resource_list.yml`.

You could always do the installation and then `ocrd resmgr download '*'` to download all models at once later on.

**Pull request merged**